src/listener/Lists.php

```diff
@@ -92,16 +92,25 @@ public function render(Lexer $lexer)
 
     /**
      * Get the html tag for the given value.
+     * 
+     * @since 1.2.0 Added exception
      *
      * @param Pick $pick
      * @return string
+     * 
+     * @throws \Exception for unknown list types
      */
     protected function getListAttribute(Pick $pick)
     {
         if ($pick->type == self::LIST_TYPE_ORDERED) {
             return 'ol';
         }
 
-        return 'ul';
+        if ($pick->type == self::LIST_TYPE_BULLET) {
+            return 'ul';
+        }
+        
+        // prevent html injection in case the attribute is user input
+        throw new \Exception('unknown list type');
```

Comment:
Please add Exception message to use statements, no inline definition. Maybe adjust exception message to be more clear and start with upper case.

```
The provided list type {$listType} is not in the list of valid tags.
```